### ISSUE TITLE
Offline Calendar

### ISSUE THREAD

**Issue by rend3r** [dmfs/opentasks#815]
Is it possible use OpenTask with offline calendars provided with https://github.com/SufficientlySecure/offline-calendar ?

**Comment by dmfs** [comment]
No, and it's not really necessary. OpenTasks has built-in support for offline task lists. You can create as many offline lists as you like. We don't have a backup-restore feature yet, but it's on the list.
Is there anything else you're missing here?

That being said, the app you mentioned could be extended with adding offline task lists to OpenTasks. I've not tried this app, so I can't tell whether this has any significant advantage over the built in offline task list support in OpenTasks.
